packages/agoric-cli/src/commands/reserve.js

```diff
@@ -1,18 +1,48 @@
 /* eslint-disable @jessie.js/no-nested-await */
 // @ts-check
 /* eslint-disable func-names */
-/* global fetch */
+/* global fetch, process */
+import { Offers } from '@agoric/inter-protocol/src/clientSupport.js';
 import { Command } from 'commander';
 import { makeRpcUtils } from '../lib/rpc.js';
-import { outputExecuteOfferAction } from '../lib/wallet.js';
+import { outputActionAndHint } from '../lib/wallet.js';
 
 /**
- *
  * @param {import('anylogger').Logger} _logger
+ * @param io
  */
-export const makeReserveCommand = async _logger => {
+export const makeReserveCommand = async (_logger, io = {}) => {
+  const { stdout = process.stdout, stderr = process.stderr, now } = io;
   const reserve = new Command('reserve').description('Asset Reserve commands');
 
+  reserve
+    .command('add')
+    .description('add collateral to the reserve')
+    .requiredOption('--give <number>', 'Collateral to give', Number)
+    .option('--collateral-brand <string>', 'Collateral brand key', 'IbcATOM')
```

Comment:
shouldn't the collateral brand be required?

Reply:
It would be a pain, and we plan to launch with just 1.

I defer to product. @otoole-brendan ?

I suppose we could do something like what cosmos-sdk does: CLI args that combine the magnitude and the denom: `1000000000ubld`. But since we know the decimalPlaces, it could be `inter bid by-price --give 5ATOM ...`.

Reply:
@dckc so I understand, wouldn't this be something we have to add anyway once we need to be able to offer CLI liquidation bidding for > 1 collateral type? (the ability to specify what collateral you are bidding for)

Reply:
we agreed to go for `--give 5IST` and such. b1e8b74a9